### PR TITLE
Add Scrollbars to Main Pages

### PR DESCRIPTION
**Pull request by seesemichaelj**
This PR adds the `react-custom-scrollbars` npm module to have better-looking scrollbars. The scrollbars have been added to the main page, wrapping everything in `ShellContainer`, as well as wrapping the `<li>` elements in `LogContainer`.

More documentation on this module can be found at https://github.com/malte-wessel/react-custom-scrollbars/tree/master/docs

Has been tested on Linux; see the below GIF.
peek 2018-05-18 10-03

### CODE REVIEW

File: src/Components/AppShell/AppShell.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
     display: flex;
     flex-grow: 1;
     justify-content: center;
-    overflow: scroll;
+    overflow: hidden;
```
Comment (benjamincburns) [inline src/Components/AppShell/AppShell.scss]: might be worth a comment here just to say that scrolling is handled by a react component

**Comment by benjamincburns** [comment]
I haven't actually ran this branch to see the UI changes, but the code looks good to me.

That said, I think you mentioned needing to do something to the layout of the Logging screen for that one to work properly? Might as well add that to this PR before merging.

**Comment by seesemichaelj** [comment]
The GIF above will show you how it looks in the UI (before the scrollbar was set to always showing).

With respect to the Logging screen, I mentioned there wasn't much we could do about the scrollbar being offset from the side due to the padding that exists on that screen. The logging screen looks a little awkward without the padding, and the scrollbar is (in my mind) acceptable to be offset by the same padding.

**Comment by benjamincburns** [comment]
In that case I'm fine if you want to merge - just to set expectations though, when you're done with all these UI changes I'll probably fire it up and review it all en masse at that point, but I suspect any changes I'd recommend at that point would be fairly minor

**Pull request merged**